### DISCUSSION TITLE
Is it possible to change the color of inactive tabs?

### DISCUSSION THREAD

**Discussion by KaBankz** [ghostty-org/ghostty#3189] [Q&A]
![CleanShot 2024-12-26 at 18 22 46@2x](https://github.com/user-attachments/assets/467aa1d2-e83f-47d8-a5c2-f1d44336c52b)

I'm trying to match ghostty with my existing iterm2 theme, and I'm 99% of the way there, except for the tab bar styling.

`macos-titlebar-style = tabs` is almost perfect, I just need the contrast/division between the tabs to be more prominent.

Also, the new tab button icon (top right) is basically invisible,
![CleanShot 2024-12-26 at 18 29 08@2x](https://github.com/user-attachments/assets/9afb6063-8cce-452f-893e-acf15ad9a662)

**Comment by Ziqi-Yang** [comment]
Same issue here. I'm using 3024 Night theme. The theme's background color is the same as the tab's font color. 

![image](https://github.com/user-attachments/assets/da36bfb1-0943-4ad7-9688-79191b961ad8)

**Reply by 00-kat** [reply]
Very late but does [`window-theme = ghostty`] help?

[`window-theme = ghostty`]: https://ghostty.org/docs/config/reference#window-theme

**Comment by somtooo** [comment]
@KaBankz  do you have your config on github. i love how your terminal looks and would love to emulate

**Reply by KaBankz** [reply]
I just pushed my ghostty config to my dotfiles repo, feel free to take a look
https://github.com/KaBankz/dotfiles/tree/dotter/ghostty

**Comment by CamilleScholtz** [comment]
Same "issue" here. TBH the iTerm tabs just look a little more refined with the colors, the border, and the slightly gray instead of bright white tab title.

**Comment by bitboxer** [comment]
Sadly on my mac I never see what tab is currently active. The color difference between active and inactive tab is just not different enough. It would be amazing to either be able to configure the color or to have better defaults ❤️
